geowebcache-webapp/src/docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY --chown=jetty:jetty . /
 USER root
 
 RUN mkdir /mnt/geowebcache_tiles /mnt/geowebcache_datadir &&                              \
-    unzip -d /var/lib/jetty/webapps/geowebcache /var/lib/jetty/webapps/geowebcache.war && \
+    unzip -d /var/lib/jetty/webapps/geowebcache /var/lib/jetty/webapps/geowebcache*.war && \
     rm -f /var/lib/jetty/webapps/geowebcache.war &&                                       \
```

Comment:
should it be something else than geowebcache.war in the docker image ? if adding a wildcard, why not on the `rm -f` statement ?

Reply:
good catch, I'm not sure how we handle or will handle the `-generic` et al suffix, hence the wildcard.

Reply:
Actually, removed the *, the less magic the better.

Reply:
normally when the docker profile is activated, we should get rid of the "-generic" suffix (I think we should get rid of it in any cases, but I think it is more of historical, when we were still using configuration profile, and the generated webapps were suffixed with the ${server} variable)